db/dcrsqlite/apisource.go

```diff
@@ -1307,11 +1307,17 @@ func (db *wiredDB) GetExplorerTx(txid string) *explorer.TxInfo {
 	return tx
 }
 
-func (db *wiredDB) GetExplorerAddress(address string, count, offset int64) *explorer.AddressInfo {
+func (db *wiredDB) GetExplorerAddress(address string, count, offset int64) (*explorer.AddressInfo, error) {
+
+	// This is here to detect a bitcoin type address
+	if (strings.HasPrefix(address, "bc") || strings.HasPrefix(address, "1") || strings.HasPrefix(address, "3")) && len(address) >= 26 {
+		return nil, fmt.Errorf("Looks like you are searching for a bitcoin address")
```

Comment:
I'm not really sure we care about this TBH.  Plus the address recognition is too basic to be correct all the time.  Just "3" would be recognized as a bitcoin address...
And how is this new error going to affect the search?  Part of the search is requesting address info.  The error returned should not stop the search from checking if it is a hash or a block height (e.g. "1").

Reply:
OK, I see @raedah's comments in the original issue, so I'm not going to fight the bitcoin address thing, but the other parts of my comment apply.  The check should be smarter about the content of the `"address"` string before declaring it a bitcoin address.  And throwing an error should not interfere with search or any other functionality.